CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Amplify for Android (Beta) - CHANGELOG
+
+A declarative library for application development using cloud services.
+
+## 0.9.0
+
+Initial release! Includes Core features, plus support for these categories:
+
+- Analytics
+- API
+- DataStore
+- Hub
```

Comment:
Hub is implemented? I saw few places this todo like this `//TODO: Dispatch to hub`

Reply:
Hub as an independent category is supported and implemented. However Hub is not used by other categories.